README.md

```diff
@@ -74,7 +74,17 @@ https://user-images.githubusercontent.com/15977946/124654387-0fd3c500-ded1-11eb-
 
 ## What's New
 
+- 2022-10-14: MMPose [v1.0.0rc0](https://github.com/open-mmlab/mmpose/releases/tag/v1.0.0rc0) is released. Major updates include:
+
+  - Support 4 light-weight pose estimation algorithms
+    - SimCC (ECCV'22): [paper](https://doi.org/10.48550/arxiv.2107.03332) | [models](https://github.com/open-mmlab/mmpose/blob/1.x/configs/body_2d_keypoint/simcc/README.md)
+    - Debias-IPR (ICCV'21): [paper](https://openaccess.thecvf.com/content/ICCV2021/papers/Gu_Removing_the_Bias_of_Integral_Pose_Regression_ICCV_2021_paper.pdf) | [models](https://github.com/open-mmlab/mmpose/blob/1.x/configs/body_2d_keypoint/integral_regression/README.md)
+    - IPR (ECCV'18): [paper](https://arxiv.org/abs/1711.08229) | [models](https://github.com/open-mmlab/mmpose/blob/1.x/configs/body_2d_keypoint/integral_regression/README.md)
+    - DSNT (ArXiv'18): [paper](https://arxiv.org/abs/1801.07372v2) | [models](https://github.com/open-mmlab/mmpose/blob/1.x/configs/body_2d_keypoint/integral_regression/README.md)
```

Comment:
BTW, the ckpt & log links of IPR, Debias-IPR and DSNT seem broken @Tau-J

Reply:
Yes, they will be available after merging dev-1.x into 1.x.

Reply:
> Yes, they will be available after merging dev-1.x into 1.x.

Sorry, I am confused. Will there be another PR to fix the links in https://github.com/open-mmlab/mmpose/blob/dev-1.x/configs/body_2d_keypoint/integral_regression/coco/resnet_debias_coco.md?